Specifications/Events/for_EventStore/when_committing_uncommitted_events.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using Dolittle.SDK.Events.for_EventConverter.given;
+using Dolittle.SDK.Execution;
+using Dolittle.SDK.Protobuf;
+using Machine.Specifications;
+using Contracts = Dolittle.Runtime.Events.Contracts;
+using It = Machine.Specifications.It;
+
+namespace Dolittle.SDK.Events.for_EventStore
+{
+    public class when_committing_uncommitted_events : given.an_event_store_and_an_execution_context
+    {
+        static UncommittedEvents uncommitted_events;
+
+        Establish context = () =>
+        {
+            var event_source_one = "e7fe623b-5fb7-4699-9b08-7c14d7556e84";
+            var event_type_one = new EventType("4134d0b4-a13f-4c5d-ae98-8e44903ab147", 2);
+            var content_one = new an_event("hello wörld", 42, true);
+            var is_public_one = true;
+
+            var event_source_two = "d3bc1b39-960b-44b4-a5f2-fa3d8c6c8056";
+            var event_type_two = new EventType("da6b65d6-1a8e-4c93-a778-5200a0b7fbbf", 1337);
+            var content_two = new an_event("bye wørld", -42, false);
+            var is_public_two = false;
+
+            var event_one = new UncommittedEvent(event_source_one, event_type_one, content_one, is_public_one);
+            var event_two = new UncommittedEvent(event_source_two, event_type_two, content_two, is_public_two);
+
+            uncommitted_events = new UncommittedEvents
+            {
+                event_one,
+                event_two,
+                event_two,
+            };
+        };
+
+        Because of = async () => await event_store.Commit(uncommitted_events);
+        It should_have_called_the_caller = () => caller.Verify(_ => _.Call(Moq.It.IsAny<EventStoreCommitMethod>(), Moq.It.IsAny<Contracts.CommitEventsRequest>(), Moq.It.IsAny<CancellationToken>()));
```

Comment:
Verify that the caller is called with the correct arguments - and also that it is not called for anything else (so we don't commit twice).